src/sentry/sentry_metrics/querying/data/execution.py

```diff
@@ -179,6 +187,34 @@ def add_group_filters(
             ),
         )
 
+    def merge_with_blocked_projects(
```

Comment:
nit: this is more of a filter (diff) than a merge

Reply:
How would you call it?

Reply:
`filter_blocked_projects`, `get_non_blocked_projects`, or something along those lines